go.mod

```diff
@@ -1,18 +1,16 @@
 module github.com/seccomp/containers-golang
 
-go 1.12
+go 1.13
 
 require (
 	github.com/blang/semver v3.5.1+incompatible // indirect
 	github.com/hashicorp/go-multierror v1.0.0 // indirect
-	github.com/opencontainers/runtime-spec v0.1.2-0.20190618234442-a950415649c7
+	github.com/opencontainers/runtime-spec v1.0.2-0.20191007145322-19e92ca81777
 	github.com/opencontainers/runtime-tools v0.9.0
-	github.com/opencontainers/selinux v1.2.2 // indirect
+	github.com/opencontainers/selinux v1.3.0 // indirect
 	github.com/seccomp/libseccomp-golang v0.9.1
 	github.com/sirupsen/logrus v1.4.2 // indirect
 	github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2 // indirect
-	github.com/xeipuuv/gojsonpointer v0.0.0-20180127040702-4e3ac2762d5f // indirect
-	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 // indirect
-	github.com/xeipuuv/gojsonschema v1.1.0 // indirect
-	golang.org/x/sys v0.0.0-20190624142023-c5567b49c5d0
+	github.com/xeipuuv/gojsonschema v1.2.0 // indirect
+	golang.org/x/sys v0.0.0-20191206220618-eeba5f6aabab
```

Comment:
This quite likely causes build issues across the stack, see https://github.com/containers/storage/commit/54d76b16f1fe. x/sys introduced breaking changes which break cross compilations. Those issues are fixed in c/storage but docker/docker breaks and that's being used quite a lot.

Before merging, would you open a PR vendoring that change into libpod or buildah? `make local-cross` would yield the error.

Reply:
Yes sure, see:
- https://github.com/containers/libpod/pull/4658
- https://github.com/containers/buildah/pull/2012

Reply:
Sticking to `golang.org/x/sys v0.0.0-20191127021746-63cb32ae39b2` for now.